woocommerce/woocommerce-gutenberg-products-block
# Store Customization > Update aspect ratio, icons and default text for the Social: Follow us on social media pattern

## What

* Add aspect ratio to the **Social: Follow us on social media** pattern to ensure the images displayed always have a square format, independent of what image the AI assigned to it. 
* Update the icon's color to match whatever is set as primary for the site. 
* Update the default text within the patterns dictionary from "Social: Follow us in social media" to "Social: Follow us on social media".

## Why



This change is necessary to ensure the design for this pattern matches the expected

Figma: Mrk6SERPZ4KrFHSjM0a8TK-fi-5716_46688

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Access the JN install with the credentials provided here > p1696409554974299-slack-C053716F2H2
2. Create a new post
3. Insert the **Social: Follow us on social media** pattern
4. Make sure it is correctly rendered both on the editor side and on the front end. The images should all be with a square format.
5. Make sure the icons have the color set as primary.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|  <img width="1163" alt="Screenshot 2023-10-06 at 10 39 06" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/4d8c4dfc-9ade-4070-89cd-1bf8bbd2f538"> | <img width="1158" alt="Screenshot 2023-10-06 at 10 50 19" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/5361b577-9c5c-4bff-9273-b564a039dc31">  |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Update aspect ratio, icons and default text for the Social: Follow us on social media pattern
